README.rst

```diff
@@ -38,9 +38,9 @@ This library:
 Prerequisites
 =============
 
-Recent versions of this library require Python (>= 2.7) to run.
+The latest versions of this library requires **Python 3** (>= 3.4).
 
-Use library version 0.14 if you're running Python 2.6.
+Use version ``0.21.x`` for Python 2.7/3.3 and ``0.14`` for Python 2.6.
```

Comment:
Would it make sense that recent versions of pip default to installing `0.21.x` if using python2.7/3.3 ?